docs/source/single_node.rst

```diff
@@ -0,0 +1,123 @@
+.. _single-node-training:
+
+
+Single-node training
+=======================
+Single-node training enables GNNs training on a very larg graphs on a single machine. This is achieved by running each worker as a separate process. At one time only one worker/process can store its data in a main memory (other processes keep their data saved on disk). SAR uses lock and a barrier for synchronizing processes. That is why you need to create them at the beginning of the program.
```

Comment:
Minor typo: "Single-node training enables `training GNNs` (swap order of GNNs and training) on a (remove `a`) very larg`e` (missing `e`)

Reply:
fixed